content/articles/svelte-with-vite-typescript-and-tailwind-css/index.md

```diff
@@ -323,7 +336,15 @@ Run your development server using `npm rum dev` and open `http://localhost:3000`
 ![new-landing-page](/engineering-education/svelte-with-vite-typescript-and-tailwind-css/new-landing-page.png)
 
 ### Conclusion
-We used TypeScript with TailwindCSS to construct a Svelte application in this post. Refer to the following links for further information about the tools and resources used.
+This guide has illustrated how you set up a TypeScript Svelte boilerplate that you can use to write TypeScript code within a Svelte application. We also configured and used TailwindCSS to create a Svelte portfolio application.
```

Comment:
Please, include links to the GitHub repo and live demo site for a better understanding.